roles/update/tasks/main.yml

```diff
@@ -36,22 +36,65 @@
   ansible.builtin.shell: |
     {{ cifmw_update_artifacts_basedir }}/control_plane_test_start.sh
 
+- name: Install plan
+  ansible.builtin.include_role:
+    name: kustomize_deploy
+    tasks_from: install_plan.yml
+  when:
+    - cifmw_ci_gen_kustomize_values_installplan_approval is defined
+    - cifmw_ci_gen_kustomize_values_installplan_approval | lower == 'manual'
```

Comment:
I realized that the change to lower was not needed, due in `roles/kustomize_deploy/tasks/install_plan.yml` it also compare with `Manual`